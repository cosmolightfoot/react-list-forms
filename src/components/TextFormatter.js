import React, { PureComponent } from 'react';
// import figlet from 'figlet';

export default class textFormatter extends PureComponent {
  state = {
    text: ''
  }

  handleChange = ({ target }) => {
    this.setState({ [target.name]: target.value });
  } 

  render() {

    return (
      <input name="text" value={text} onChange={this.handleChange} />
    );
  }
}
