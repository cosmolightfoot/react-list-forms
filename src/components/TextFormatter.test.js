import React from 'react';
import { shallow } from 'enzyme';
import TextFormatter from './TextFormatter.js';

describe('Text Formatter test', () => {
  it('Renders the text formatter', () => {
    const wrapper = shallow(<TextFormatter />);
    expect(wrapper).toMatchSnapshot();
  });
});
