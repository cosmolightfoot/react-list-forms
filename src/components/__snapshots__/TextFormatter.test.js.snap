// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Formatter test Renders the text formatter 1`] = `
<Fragment>
  <select
    name="font"
    onChange={[Function]}
    value=""
  >
    <option
      key="Alligator"
      value="Alligator"
    >
      Alligator
    </option>
    <option
      key="Alpha"
      value="Alpha"
    >
      Alpha
    </option>
    <option
      key="Barbwire"
      value="Barbwire"
    >
      Barbwire
    </option>
    <option
      key="Banner4"
      value="Banner4"
    >
      Banner4
    </option>
    <option
      key="Isometric3"
      value="Isometric3"
    >
      Isometric3
    </option>
  </select>
  <input
    name="text"
    onChange={[Function]}
    value=""
  />
  <input
    name="color"
    onChange={[Function]}
    type="color"
    value="black"
  />
  <pre
    style={
      Object {
        "color": "black",
      }
    }
  />
</Fragment>
`;
